orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/RollingRedBlackStrategy.groovy

```diff
@@ -56,7 +56,7 @@ class RollingRedBlackStrategy implements Strategy, ApplicationContextAware {
   DetermineTargetServerGroupStage determineTargetServerGroupStage
 
   @Autowired
-  TargetServerGroupResolver targetServerGroupResolver
+  SourceResolver sourceResolver
```

Comment:
Doesn't look like this is actually used anywhere?

Reply:
thanks!